playgrounds/all-field-types.mongodb

```diff
@@ -0,0 +1,42 @@
+const { Buffer } = require('node:buffer');
+
+use('test');
+
+// Create a document with all the BSON types.
+const doc = {
+  double: new Double(1.2), // Double, 1, double
+  string: 'Hello, world!', // String, 2, string
+  object: { key: 'value' }, // Object, 3, object
+  array: [1, 2, 3], // Array, 4, array
+  binData: new Binary(Buffer.from([1, 2, 3])), // Binary data, 5, binData
+  // Undefined, 6, undefined (deprecated)
+  objectid: new ObjectId(), // ObjectId, 7, objectId
```

Comment:
Do you mind changing this to objectId? It is a small thing, but I'm trying to match the aliases as listed in the documentation https://www.mongodb.com/docs/manual/reference/bson-types/

(things aren't consistently named everywhere and I'm trying to standardise on something)